time stamp
string data_msg
float32 data_x
float32 data_y